src/cascadia/TerminalSettingsEditor/ObjectModel/Profile.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include "Microsoft.Terminal.Settings.Model.Profile.g.h"
+#include "Utils.h"
+
+namespace winrt::Microsoft::Terminal::Settings::Model::implementation
+{
+    struct Profile : ProfileT<Profile>
+    {
+    public:
+        Profile() = default;
+
+        WINRT_CALLBACK(PropertyChanged, Windows::UI::Xaml::Data::PropertyChangedEventHandler);
+        OBSERVABLE_GETSET_PROPERTY(guid, Guid, _PropertyChangedHandlers);
+        OBSERVABLE_GETSET_PROPERTY(hstring, Name, _PropertyChangedHandlers, L"Default");
+        OBSERVABLE_GETSET_PROPERTY(hstring, Source, _PropertyChangedHandlers);
+        OBSERVABLE_GETSET_PROPERTY(guid, ConnectionType, _PropertyChangedHandlers);
+        OBSERVABLE_GETSET_PROPERTY(hstring, Icon, _PropertyChangedHandlers);
+        OBSERVABLE_GETSET_PROPERTY(bool, Hidden, _PropertyChangedHandlers, false);
+        OBSERVABLE_GETSET_PROPERTY(CloseOnExitMode, CloseOnExit, _PropertyChangedHandlers, CloseOnExitMode::graceful);
+        OBSERVABLE_GETSET_PROPERTY(hstring, TabTitle, _PropertyChangedHandlers);
+
+        // Terminal Control Settings
+        OBSERVABLE_GETSET_PROPERTY(bool, UseAcrylic, _PropertyChangedHandlers, false);
+        OBSERVABLE_GETSET_PROPERTY(double, AcrylicOpacity, _PropertyChangedHandlers, 0.5);
+        OBSERVABLE_GETSET_PROPERTY(ScrollbarState, ScrollState, _PropertyChangedHandlers, ScrollbarState::visible);
+        OBSERVABLE_GETSET_PROPERTY(hstring, FontFace, _PropertyChangedHandlers, L"Cascadia Mono");
+        OBSERVABLE_GETSET_PROPERTY(int32_t, FontSize, _PropertyChangedHandlers, 12);
+        OBSERVABLE_GETSET_PROPERTY(Windows::UI::Text::FontWeight, FontWeight, _PropertyChangedHandlers);
+        OBSERVABLE_GETSET_PROPERTY(hstring, Padding, _PropertyChangedHandlers, L"8, 8, 8, 8");
+        OBSERVABLE_GETSET_PROPERTY(bool, CopyOnSelect, _PropertyChangedHandlers, false);
+        OBSERVABLE_GETSET_PROPERTY(hstring, Commandline, _PropertyChangedHandlers, L"cmd.exe");
+        OBSERVABLE_GETSET_PROPERTY(hstring, StartingDirectory, _PropertyChangedHandlers, L"%USERPROFILE%");
+        OBSERVABLE_GETSET_PROPERTY(hstring, EnvironmentVariables, _PropertyChangedHandlers);
+        OBSERVABLE_GETSET_PROPERTY(hstring, BackgroundImage, _PropertyChangedHandlers);
+        OBSERVABLE_GETSET_PROPERTY(double, BackgroundImageOpacity, _PropertyChangedHandlers, 0.5);

```

Comment:
Shouldn't some of these defaults load out of `defaults.json` instead of having another matching definition of them here?